easybuild/easyconfigs/d/Doxygen/Doxygen-1.8.10-intel-2015b.eb

```diff
@@ -0,0 +1,19 @@
+name = 'Doxygen'
+version = '1.8.10'
+
+homepage = 'http://www.doxygen.org'
+description = """Doxygen is a documentation system for C++, C, Java, Objective-C, Python, 
+ IDL (Corba and Microsoft flavors), Fortran, VHDL, PHP, C#, and to some extent D."""
+
+toolchain = {'name': 'intel', 'version': '2015b'}
+
+sources = ['%(namelower)s-%(version)s.src.tar.gz']
+source_urls = ['http://ftp.stack.nl/pub/users/dimitri/']
+
+builddependencies = [
+    ('CMake', '3.4.1'),
```

Comment:
it is, but it's a build dep, so doesn't matter

we really need support for something like `'CMake', '>= 3.4'` for this, but it's not there yet, and we tend to stick to the last version of deps if we install a new software version, so...